Makefile

```diff
@@ -132,5 +139,25 @@ tag-release:
 	hack/tag-release.sh $(VERSION)
 	$(IMGTOOL) push -q "$(IMAGE)" "$(IMG):latest"
 
+.PHONY: shellcheck
```

Comment:
Move to shell script hack/download_shellcheck.sh, no need to check for existence this make rule will only be invoked if shellcheck is missing